### PR TITLE
Table: add header-align, Fixed #1424

### LINKED ISSUE #1424: [Table] 建议增加 Header 的 align 属性
现有的align属性设置后header和cell的对齐是一致的，建议增加column header的align属性，可以分别设置

### PR DESCRIPTION
**Pull request by Kingwl**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

#1424

### CODE REVIEW

File: examples/docs/zh-CN/table.md [hunk 1]
```diff
@@ -1421,6 +1421,7 @@
 | context | 设置上下文环境，例如设置当前上下文就是 `_self`，父级就是 `$parent`，根组件 `$root` | Object | - | Table 所处上下文 |
 | inline-template | 指定该属性后可以自定义 column 模板，参考多选的时间列，通过 row 获取行信息。总共可以获取到 `{ row(当前行), column(当前列), $index(行数), store(table store) }` 以及 Table 所处的上下文环境。 | — | — |
 | align | 对齐方式 | String | left, center, right | left |
+| header_align | 表头对齐方式,若不设置该项，则使用表格的对齐方式 | String | left, center, right | — |
```
Comment (furybean) [inline examples/docs/zh-CN/table.md]: ……

Comment (furybean) [inline examples/docs/zh-CN/table.md]: 感觉楼上两个 approve 的根本没 review，这个笔误太明显了。

Comment (Leopoldthecoder) [inline examples/docs/zh-CN/table.md]: 我来改，顺便补英文文档

Comment (Kingwl) [inline examples/docs/zh-CN/table.md]: 🌚

**Pull request merged**